safe/definitions/utilities.py

```diff
@@ -4,7 +4,9 @@
 from collections import OrderedDict
 from copy import deepcopy
 from os import listdir
-from os.path import join, exists, splitext
+from os.path import join, exists, splitext, split
+from PyQt4 import QtXml
+from PyQt4.QtXml import QDomNode
```

Comment:
Perhaps import specific module. You import QDomNode (in QtXml) but also import QtXml above it.